### ISSUE TITLE
The "Beta" is not clear on "High Contrast On" mode'

### ISSUE THREAD

**Issue by Helen09** [microsoft/pxt-ev3#661]
**Repro steps:**
• Open https://makecode.mindstorms.com/beta#editor
• Click "More..." -> "High Contrast On" to open high contrast mode
• Hover on the top left corner of the page
**Expected result:**
The "Beta" should be clear 
**Actual result:**
The "Beta" is not clear on "High Contrast On" mode'
![1234](https://user-images.githubusercontent.com/23466737/40533412-3e2a2036-5ff2-11e8-854c-a393732f7552.gif)
